### PR TITLE
Fix or remove dead links caught by awesome_bot

### PR DESCRIPTION
**Pull request by gertig**
Fix or remove the links that `awesome_bot` identified as issues.

From the Travis CI build:

> Links 
  1. [L047] 404 https://github.com/tadast/switching-to-contracting-uk/tree/formationsfactory  
  2. [L097] 404 https://m.signalvnoise.com/why-i-work-remotely-hint-it-has-nothing-to-do-with-productivity-34ace30f74fc?gi=39b26e8f40a1  
  3. [L170]  https://remotelist.io/ Cannot assign requested address - connect(2) for "remotelist.io" port 443 
  4. [L202]  http://blackbox.vc Connection timed out - connect(2) for "blackbox.vc" port 80 
  5. [L224] 404 https://legislature.vermont.gov/assets/Documents/2018/Docs/BILLS/S-0094/S-0094%20As%20Passed%20by%20Both%20House%20and%20Senate%20Unofficial.pdf  
  6. [L245]  http://workingnomads.co/ Connection reset by peer 
  7. [L320] 404 https://findify.io/careers/  
  8. [L452] 404 https://zamphyr.com/posao/

### CODE REVIEW

**Comment by gertig** [comment]
Hi @RichardLitt just checking if there was anything I missed in this PR?

File: README.md [hunk 1]
```diff
@@ -167,7 +166,6 @@ A curated list of awesome [remote working](https://en.wikipedia.org/wiki/Telecom
   1. [remote-jobs](https://github.com/jessicard/remote-jobs) - A list of semi to fully remote-friendly companies in tech
   1. [Remotees](https://remotees.com/)
   1. [Remote.co Jobs](https://remote.co/remote-jobs/)
-  1. [RemoteList](https://remotelist.io/) - Has great job category filters
```
Comment (lukasz-madon) [inline README.md]: seems to be working

File: README.md [hunk 2]
```diff
@@ -241,7 +238,6 @@ A curated list of awesome [remote working](https://en.wikipedia.org/wiki/Telecom
   1. [Remote Report](http://remote.report) - A collection of the best remote, support focused and non-technical jobs delivered weekly
   1. [Remoteur](http://www.remoteur.com) - Remote jobs in Europe delivered to your inbox bi-weekly
   1. [Remotive - productive remote workers](http://remotive.io/) - A weekly newsletter on Remote Tips & Jobs sent to 10,000+ Remote Workers
-  1. [Working Nomads](http://workingnomads.co/) - Job posts
```
Comment (lukasz-madon) [inline README.md]: this one works as well (301)

File: README.md (hunk 1 above)
Comment (gertig) [reply README.md]: That's strange, just tried it again and can't get it to load.

File: README.md (hunk 2 above)
Comment (gertig) [reply README.md]: Ahh, yes, I think I removed this one as a duplicate / miscategorized. Sorry, should have done in a separate PR.

Comment (gertig) [reply README.md]: Want me to leave this one in and resubmit?

File: README.md (hunk 1 above)
Comment (lukasz-madon) [reply README.md]: works only in FF. We can remove it then.

**Pull request merged**